Automattic/wp-calypso
# Adding WP Courses task to My Home.

#### Changes proposed in this Pull Request
* This PR contains the client side code for the WP Courses task card on My Home.
* The WPcom patch is at D53238-code
* More details about this PR are available at pau2Xa-2ix-p2

Here's a screenshot of the card in place:
<img width="1431" alt="Screen Shot 2020-11-23 at 12 28 49 PM" src="https://user-images.githubusercontent.com/35781181/100160318-52361d80-2e7d-11eb-996f-d367988a72b7.png">


#### Testing instructions
* Patch your sandbox with D53238.
* Checkout this branch.
* Point API calls to your sandbox.
* Force this view to load by accessing http://calypso.localhost:3000/home/YOURSITESLUG?dev=true&view=VIEW_WP_COURSES
* Create a new account and go through the My Home task cards and confirm that this view shows as expected.
* Click the link and confirm that you're taken to the WP Courses home page.
